.github/workflows/upgrade.yml

```diff
@@ -92,3 +92,5 @@ jobs:
           author: github-actions <github-actions@github.com>
           committer: github-actions <github-actions@github.com>
           signoff: true
+    environment:
+      name: protected-main-env
```

Comment:
This file is generated. I modified `.projenrc.js` so these two lines can be added.